db/seeds.rb

```diff
@@ -5,3 +5,83 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+submission_email = ENV["EMAIL"] || `git config --get user.email`.strip
+organisation_id = 1 # Assumes you're using the default database seed for forms-admin
+
+all_question_types_form = Form.create!(
+  name: "All question types form",
+  pages: [
+    Page.create(
+      question_text: "Single line of text",
+      answer_type: "text",
+      answer_settings: {
+        input_type: "single_line",
+      },
+    ),
+    Page.create(
+      question_text: "Number",
+      answer_type: "number",
+    ),
+    Page.create(
+      question_text: "Address",
+      answer_type: "address",
+      answer_settings: {
+        input_type: {
+          international_address: false,
+          uk_address: true,
+        },
+      },
+    ),
+    Page.create(
+      question_text: "Email address",
+      answer_type: "email",
+    ),
+    Page.create(
+      question_text: "Todays Date",
+      answer_type: "date",
+      answer_settings: {
+        input_type: "other_date",
+      },
+    ),
+    Page.create(
+      question_text: "National Insurance number",
+      answer_type: "national_insurance_number",
+    ),
+    Page.create(
+      question_text: "Phone number",
+      answer_type: "phone_number",
+    ),
+    Page.create(
+      question_text: "Selection from a list of options",
+      answer_type: "selection",
+      answer_settings: {
+        "only_one_option": "0", # TODO: investigate why we set this to "0"
```

Comment:
I've created a ticket for this: https://trello.com/c/prWyiTVd/1310-forms-admin-saves-onlyoneoption-0-into-database-if-user-does-not-check-box

Reply:
I think I probably introduced this - I vaguely remember running into trouble with these boolean fields and ActiveResource. There's almost certainly a better way of doing it, so I appreciate you adding the card to investigate it!